BrowserKit/Sources/ComponentLibrary/BottomSheet/BottomSheetViewController.swift

```diff
@@ -90,7 +88,10 @@ public class BottomSheetViewController: UIViewController,
         super.viewDidLoad()
         sheetView.alpha = 1
         setupChildViewController()
-        closeButton.accessibilityLabel = viewModel.closeButtonA11yLabel
+
+        let closeButtonViewModel = CloseButtonViewModel(a11yLabel: viewModel.closeButtonA11yLabel,
+                                                        a11yIdentifier: "a11yCloseButton")
```

Comment:
I think the a11y identifier should come from the viewModel as well and be added to the `AccessibilityIdentifiers` struct.

Reply:
Sounds good. Let's do that. Thanks @cyndichin 🙏